### ISSUE TITLE
warnings during training

### ISSUE THREAD

**Issue by Ocean23333** [kangben258/MCITrack#6]
Hi, i tried to train the model on got-10k*.
I modified the batch size to 32 and run the script below.
torchrun --nproc_per_node 4 lib/train/run_training.py --script mcitrack --config mcitrack_b224_got --save_dir .
Training appeared to be ongoing but these warnings appeared:
UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [256, 768, 1, 1], strides() = [768, 1, 768, 768]
bucket_view.sizes() = [256, 768, 1, 1], strides() = [768, 1, 1, 1] (Triggered internally at ../torch/csrc/distributed/c10d/reducer.cpp:320.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
data sampler bug: Input and output sizes should be greater than 0, but got input (H: 0, W: 186) output (H: 112, W: 112)

Did i do something wrong？Or is it just like that?

**Comment by kangben258** [comment]
What version of PyTorch are you using? If it is too low, you can try upgrading; or you can ignore this warning, it seems have no impact on performance

**Comment by Ocean23333** [comment]
I am using the Version: 2.1.2+cu118, as reported in the paper.When i set up the environment by running 'bash install.sh', a pytorch of version 2.5.1 was got. I then uninstalled that version and installed the current one.
It do have no impact on performance.i shoud better ignore it XD.
It's a great job！
